clang/include/clang/Sema/Sema.h

```diff
@@ -11311,6 +11311,10 @@ class Sema final {
                         OpenMPDirectiveKind &Kind,
                         OpenMPDirectiveKind &PrevMappedDirective);
 
+  /// [target] teams loop is equivalent to parallel for if associated loop
+  /// nest meets certain critera.
+  bool teamsLoopCanBeParallelFor(Stmt *Astmt);
+
```

Comment:
Do you really need to expose it in Sema or you can make it just a static local function in SemaOpenMP.cpp?

Reply:
> Do you really need to expose it in Sema or you can make it just a static local function in SemaOpenMP.cpp?

Actually, no. Thanks for noticing that. I'll make the change.